UI_UX/Frontend/src/styles/AppHeader.scss

```diff
@@ -1,21 +1,81 @@
-$headerHeight: 74px;
+$headerHeight: 90px;
 
 .AppHeader {
 	height: $headerHeight;
-	background-color: #200344;
+	background-image: linear-gradient(to right, #0b3f9c, #2e58db);;
```

Comment:
Let's start using semantic names. We use two names, color and usage (see https://github.com/MyBitFoundation/MyBit.io/blob/master/styles/colors.scss for reference).
```
$white: #fff;
$text-color: $white;
```

Reply:
Also seems you have an extra semicolon :eyes: